results/net48/MessageTemplateCacheBenchmark_Cached.md

```diff
@@ -0,0 +1,83 @@
+| Concurrent |    10 |                      1 |   NA |    NA |     ? |       ? |
+|            |       |                        |      |       |       |         |
+| **Dictionary** |    **20** |                     **-1** |   **NA** |    **NA** |     **?** |       **?** |
+|  Hashtable |    20 |                     -1 |   NA |    NA |     ? |       ? |
+| Concurrent |    20 |                     -1 |   NA |    NA |     ? |       ? |
+|            |       |                        |      |       |       |         |
+| **Dictionary** |    **20** |                      **1** |   **NA** |    **NA** |     **?** |       **?** |
+|  Hashtable |    20 |                      1 |   NA |    NA |     ? |       ? |
+| Concurrent |    20 |                      1 |   NA |    NA |     ? |       ? |
+|            |       |                        |      |       |       |         |
+| **Dictionary** |    **50** |                     **-1** |   **NA** |    **NA** |     **?** |       **?** |
+|  Hashtable |    50 |                     -1 |   NA |    NA |     ? |       ? |
+| Concurrent |    50 |                     -1 |   NA |    NA |     ? |       ? |
+|            |       |                        |      |       |       |         |
+| **Dictionary** |    **50** |                      **1** |   **NA** |    **NA** |     **?** |       **?** |
+|  Hashtable |    50 |                      1 |   NA |    NA |     ? |       ? |
+| Concurrent |    50 |                      1 |   NA |    NA |     ? |       ? |
+|            |       |                        |      |       |       |         |
+| **Dictionary** |   **100** |                     **-1** |   **NA** |    **NA** |     **?** |       **?** |
+|  Hashtable |   100 |                     -1 |   NA |    NA |     ? |       ? |
+| Concurrent |   100 |                     -1 |   NA |    NA |     ? |       ? |
+|            |       |                        |      |       |       |         |
+| **Dictionary** |   **100** |                      **1** |   **NA** |    **NA** |     **?** |       **?** |
+|  Hashtable |   100 |                      1 |   NA |    NA |     ? |       ? |
+| Concurrent |   100 |                      1 |   NA |    NA |     ? |       ? |
+|            |       |                        |      |       |       |         |
+| **Dictionary** |  **1000** |                     **-1** |   **NA** |    **NA** |     **?** |       **?** |
+|  Hashtable |  1000 |                     -1 |   NA |    NA |     ? |       ? |
+| Concurrent |  1000 |                     -1 |   NA |    NA |     ? |       ? |
+|            |       |                        |      |       |       |         |
+| **Dictionary** |  **1000** |                      **1** |   **NA** |    **NA** |     **?** |       **?** |
+|  Hashtable |  1000 |                      1 |   NA |    NA |     ? |       ? |
+| Concurrent |  1000 |                      1 |   NA |    NA |     ? |       ? |
+
+Benchmarks with issues:
```

Comment:
Seems like some of the benchmarks are failing - any chance this is related to the PR? There are a couple more results with "issues" sections like this one.